src/locales/cy/translation/confirmation-person-removed-themselves.json

```diff
@@ -0,0 +1,7 @@
+{
+    "you_are_no_longer_authorised": "[CY] You are no longer digitally authorised to file online for ",
+    "what_happens_now": "[CY] What happens now you've removed your digital authorisation for this company",
+    "weve_sent_an_email": "[CY] We’ve sent an email to the company’s registered email address and to any other people who are authorised to let them know.",
+    "go_to_your_companies": "[CY] Go to your companies",
+    "button_aria_label": "[CY] Button to proceed to the 'your companies' page"
```

Comment:
Guessing [CY] is just a placeholder till we get translations?

Reply:
Exactly, these are just placeholders until we get the Welsh translations.